google-cloud-pubsub/src/main/java/com/google/cloud/pubsub/v1/Publisher.java

```diff
@@ -141,6 +141,9 @@ private Publisher(Builder builder) throws IOException {
     if (channelProvider.needsHeaders()) {
       channelProvider = channelProvider.withHeaders(builder.headerProvider.getHeaders());
     }
+    if (channelProvider.needsEndpoint()) {
```

Comment:
From a review perspective, I think it would be good to put the manual changes in a separate PR, although then it may be impossible to get tests to pass?

Reply:
Correct.

Maybe one process we can follow in the future is to include two commits in the PR - one with only autogen changes, and the next with the manual fixups.